firestore.rules

```diff
@@ -0,0 +1,12 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read, write: if false;
+    }
+    match /faqs/{docId} {
+      allow read: if true;
```

Comment:
These aren't good firestore rules, so maybe make a GitHub issue so we know to fix these rules later on.